sklearn/calibration.py

```diff
@@ -89,6 +90,24 @@ class CalibratedClassifierCV(BaseEstimator, ClassifierMixin,
         .. versionchanged:: 0.22
             ``cv`` default value if None changed from 3-fold to 5-fold.
 
+    ensemble : bool, default=True
+        Determines how the calibrator is fit, if `cv` is not `'prefit'`.
```

Comment:
We prefer `fitted` to avoid any ambiguity

```suggestion
        Determines how the calibrator is fitted, if `cv` is not `'prefit'`.
```

(there are 2 other occurrences)